APIs/OpenSearch.md

```diff
@@ -150,4 +153,68 @@ The resulting XML file provides full list of the parameters for the collection,
 For comprehensive Order API description you can visit these pages:
 was already in comment -- [**https://finder.creodias.eu/api/docs/**](https://finder.creodias.eu/api/docs/) --
 
-[**EO Data Ordering API2 Manual**](https://creodias.eu/-/comletiondatestartdate-and-in-finder-api-v2?) -->
\ No newline at end of file
+[**EO Data Ordering API2 Manual**](https://creodias.eu/-/comletiondatestartdate-and-in-finder-api-v2?) -->
+
+
+## Product Download
+
+For downloading products you need an authorization token as only authorized users are allowed to download data products.
+
+
+To get the token you can use the following scripts:
+
+```bash
+curl --location --request POST 'https://identity.dataspace.copernicus.eu/auth/realms/CDSE/protocol/openid-connect/token' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'grant_type=password' \
```

Comment:
Googling for password grant reveals that it's insecure, so not sure why we would recommend it. oidc-agent would be a more secure tool and also easier to use?
`The Password grant type is a legacy way to exchange a user's credentials for an access token. Because the client application has to collect the user's password and send it to the authorization server, it is not recommended that this grant be used at all anymore.`

Reply:
logged a separate issue for this